task.py

```diff
@@ -265,13 +265,18 @@ def get_namespace(self) -> str:
     def get_duration(self) -> int:
         return self.ts.cfg_descr.get_tft().duration
 
+    def get_oci_bin(self) -> int:
+        return self.ts.cfg_descr.get_tft().oci_bin
+    
     def get_template_args(self) -> dict[str, str]:
         return {
             "name_space": self.get_namespace(),
             "test_image": tftbase.get_tft_test_image(),
             "image_pull_policy": tftbase.get_tft_image_pull_policy(),
-            "command": "/sbin/init",
-            "args": "",
+            # /sbin/init is not working with containerd. The pod goes into CrashLoopBackOff state 
```

Comment:
Could you try with  https://github.com/wizhaoredhat/ocp-traffic-flow-tests/pull/95

Reply:
I also think #95 is the solution.

This runs `/sbin/init` (that is, systemd) as non-PID 1. While there are cases where that is desirable (`--user`), I think it's wrong here.

Reply:
#95 Was merged. Could you fix the conflicting files and see if it fixes it for you.

Reply:
Merging with #95 fixed this. Thanks!